data/data/bootstrap/openstack/files/etc/NetworkManager/dispatcher.d/30-local-dns-prepender

```diff
@@ -6,13 +6,27 @@ case "$STATUS" in
     logger -s "NM local-dns-prepender triggered by ${1} ${2}."
     DNS_IP="127.0.0.1"
     set +e
-    logger -s "NM local-dns-prepender: Checking if local DNS IP is the first entry in resolv.conf"
-    if grep nameserver /etc/resolv.conf | head -n 1 | grep -q "$DNS_IP" ; then
-        logger -s "NM local-dns-prepender: local DNS IP already is the first entry in resolv.conf"
-        exit 0
+    if systemctl -q is-enabled systemd-resolved; then
```

Comment:
Outside of the scope of this PR, but I wonder if it would be better to have a single instance of these files and link to that single instance from the various platform directories. That way we don't have to worry about making the identical changes in 4 places.

Reply:
This file is shared with MCO and previously we used https://github.com/openshift/baremetal-runtimecfg/ to share scripts (like [non-virtual-ip](https://github.com/openshift/baremetal-runtimecfg/pull/46)). Perhaps we could use this repo as a script storage and extract it from runtimecfg image in installer/MCO?